### DISCUSSION TITLE
Pinecone New Serverless API

### DISCUSSION THREAD

**Discussion by 1J07** [langflow-ai/langflow#1374] [General]
Hi, looks like langflow isn't written for use with serverless index. Trying to use Langflow and pinecone for PDF uploader and GenAI.
Any comments or links on how to do this on serverless index? I want to avoid the costs of Pinecone Pods.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
